packages/itwinui-css/src/dialog/dialog.scss

```diff
@@ -3,6 +3,7 @@
 @import '../utils/index';
 
 $iui-dialog-width: 380px;
+$iui-dialogue-resizable-min-height: 130px;
```

Comment:
Value looks a bit random. How did you come up with `130px`?

Reply:
Just by testing around and see which ones look like the best min-height on my screen, which I have included a screenshot in the description. Also I thought the width right above looked random too so I thought that was also just from eye-gauging. This is something that I wanted input on per the PR description since hard coded value is always risky considering it doesn't take into account different screen, browsers or OS.

Reply:
Let's look at it mathematically then. If we add up all the individual parts that make up the UI, then get somewhere around ~124px, which can be converted to rem to get 7.75rem.
![](https://user-images.githubusercontent.com/9084735/216631233-7579f216-7ac9-4694-a7ab-fa629f1a2692.png)

```suggestion
$iui-dialogue-resizable-min-height: 7.75rem;
```

It would be a bit verbose to do all those calculations in code so you could just add a comment.

```suggestion
// 7.75rem = 124px = title bar height + margins + button height + partially visible text
$iui-dialogue-resizable-min-height: 7.75rem;
```

Reply:
That makes sense. My question would be why do we want to have the leftover part be 16px? Is it universal rule to have that be less than line-height? Or a range of line-height > x > 0 where x is leftover would be adequate?

Reply:
We want it to be less than the line-height so that the text is partially visible, which serves an additional indicator that the region in scrollable. We use a similar approach in menu-items
![](https://user-images.githubusercontent.com/9084735/216655121-fb86b5e7-8f3f-400c-9cc1-4b5348242ab2.png)